### PR TITLE
Move auth and network tests to use framework/log

### PR DESCRIPTION
**Pull request by johnSchnake**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
This is part of the transition to using framework/log instead
of the Logf inside the framework package. This will help with
import size/cycles when importing the framework or subpackages.

**Which issue(s) this PR fixes**:
Ref #77359 

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

/sig testing
/area test
/area test-framework

### CODE REVIEW

File: test/e2e/auth/audit.go [hunk 1]
```diff
@@ -35,6 +35,7 @@ import (
 	clientset "k8s.io/client-go/kubernetes"
 	restclient "k8s.io/client-go/rest"
 	"k8s.io/kubernetes/test/e2e/framework"
+	e2elog "k8s.io/kubernetes/test/e2e/framework"
```
Comment (liggitt) [inline test/e2e/auth/audit.go]: this is importing k8s.io/kubernetes/test/e2e/framework under a different name, not using k8s.io/kubernetes/test/e2e/framework/log as I expected

Comment (liggitt) [reply test/e2e/auth/audit.go]: same comment applies to all places

Comment (johnSchnake) [reply test/e2e/auth/audit.go]: Ugh great catch, sorry about. Just a copy/paste error everywhere; my eyes never finished reading the line I guess. Patch coming.

**Pull request merged**